### DISCUSSION TITLE
Where is Go language upgrade?

### DISCUSSION THREAD

**Discussion by setanarut** [Homebrew/discussions#2453] [Everyday usage] [answered]
GO is outdated.
https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/go.rb
There is newer version of go (1.17.3) -> https://golang.org/dl/#stable
```
> brew upgrade go
Warning: go 1.17.2 already installed
```

**Comment by carlocab** [comment, marked as answer]
Open pull requests are usually a good place to look: https://github.com/Homebrew/homebrew-core/pull/88775
